inductiva/tasks/methods.py

```diff
@@ -45,7 +45,7 @@ def to_dict(list_of_tasks: Iterable[Task]) -> Mapping[str, List[Any]]:
         if executer is None:
             resource_type = None
         else:
-            resource_type = executer["vm_type"]
+            resource_type = executer["host_type"] + " " + executer["vm_type"]
```

Comment:
I've seen this in multiple places (starting at the backend). It should be `executor` and not `executer` 😉

Reply:
As far as I can tell, _executer_ doesn't exist [in the Cambridge dictionary](https://dictionary.cambridge.org/spellcheck/english/?q=executer). The [Oxford one](https://www.oed.com/dictionary/executor_n?tab=meaning_and_use#4988318) points to an _executor_ as being  _an agent, doer_, and the same as executer. [Wikipedia](https://en.wikipedia.org/wiki/Executer) points to a band.
All uses of the _execut*r_ expression in tech refer to _executor_.